### PR TITLE
Add xTB as a valid functional to the workflow

### PR DESCRIPTION
**Pull request by lipelopesoliveira**
This PR adds the `xTB` to the list of available functional to optimize the structure and calculate the vibrational spectra.

### CODE REVIEW

File: bin/raman_ir.py [hunk 1]
```diff
@@ -122,7 +122,7 @@
                     default='PBE',
                     action='store',
                     required=False,
-                    choices=['PBE', 'xTB'],
+                    choices=['PBE', 'XTB'],
```
Comment (neumannrf) [inline bin/raman_ir.py:125]: Is there a difference in meaning between `xTB` and `XTB`?

File: bin/raman_ir.py [hunk 2]
```diff
@@ -229,59 +229,76 @@
...
-                        "reference_functional": arg.Functional,
-                        "r_cutoff": 16,
-                        "parameter_file_name": f"{arg.CP2KDataDir}/dftd3.dat"
+            "stress_tensor": "analytical"
+        }
+
+if arg.Functional == 'XTB':
+    Force_Eval_Dict['+dft']['+qs'] = {
+                    'method': 'xTB',
+                    '+xTB': {
```
Comment (neumannrf) [inline bin/raman_ir.py]: Is there a difference in meaning between `xTB` and `XTB`?

**Review by neumannrf — COMMENTED** [review]
Thanks, @lipelopesoliveira ! 

I just had one minor question concerning the distinction between `xTB` and `XTB`.

**Comment by lipelopesoliveira** [comment]
Sorry @neumannrf , my mistake. The keywords are not case-sensitive, but but I decide to leave everything capitalized to maintain consistency. I must have hit an extra `ctrl + z` and ended up undoing the capitalization by mistake.

**Review by neumannrf — APPROVED** [review]
Thank you for the clarification, @lipelopesoliveira !!!

**Pull request merged**